lib/locales/el.ts

```diff
@@ -0,0 +1,501 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'el',
+
+  seasons: {
+    advent: {
+      season: 'Περίοδος της Παρουσίας',
+      weekday: '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας της Παρουσίας',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της Παρουσίας',
+      privileged_weekday: '{{day}} $t(months:11)',
+    },
+
+    christmas_time: {
+      season: 'Περίοδος των Χριστουγέννων',
+      day: '$t(weekdays:{{dow}}) των Χριστουγέννων',
+      octave: '{{count}}η ημέρα της Ογδοάδας των Χριστουγέννων',
+      second_sunday_after_christmas: 'Δεύτερη Κυριακή μετά τα Χριστούγεννα',
+      before_epiphany: '{{day}} $t(months:0)',
+      after_epiphany: '$t(weekdays:{{dow}}) μετά την πανήγυρη των Θεοφανίων',
+    },
+
+    ordinary_time: {
+      season: 'Kοινή περίοδος του έτους',
+      weekday:
+        '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "genitive" }) της κοινής περίοδου του έτους',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της κοινής περίοδου του έτους',
+    },
+
+    lent: {
+      season: 'Περίοδος της αγίας Τεσσαρακοστής',
+      weekday:
+        '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας της Τεσσαρακοστής',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της Τεσσαρακοστής',
+      day_after_ash_wed: '$t(weekdays:{{dow}}) των Τεφρών',
+      holy_week_day: 'Μεγάλη $t(weekdays:{{dow}})',
+    },
+
+    paschal_triduum: { season: 'Πασχαλινό τριήμερο' },
+
+    easter_time: {
+      season: 'Περίοδος του Πάσχα',
+      weekday: '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας του Πάσχα',
+      sunday: '$t(ordinals:{{week}}) Κυριακή του Πάσχα',
+      octave: '$t(weekdays:{{dow}}) της Διακαινησίμου',
+    },
+  },
+
+  ranks: {
+    solemnity: 'πανήγυρη',
+    feast: 'εορτή',
+    memorial: 'μνήμη',
+    sunday: 'Κυριακή',
+    weekday: 'καθημερινή',
+  },
+
+  cycles: {
+    proper_of_time: 'Ιδιαίτερο του περιόδου',
+    proper_of_saints: 'Ιδιαίτερο των αγίων',
+    sunday_year_a: 'Κύκλος Α΄',
+    sunday_year_b: 'Κύκλος Β΄',
+    sunday_year_c: 'Κύκλος Γ΄',
+    weekday_year_1: 'Κύκλος Α΄',
+    weekday_year_2: 'Κύκλος Β΄',
+    psalter_week_1: 'Εβδομάδα Αʹ',
+    psalter_week_2: 'Εβδομάδα Β΄',
+    psalter_week_3: 'Εβδομάδα Γ΄',
+    psalter_week_4: 'Εβδομάδα Δ΄',
+  },
+
+  weekdays: {
+    0: 'Κυριακή',
+    1: 'Δευτέρα',
+    2: 'Τρίτη',
+    3: 'Τετάρτη',
+    4: 'Πέμπτη',
+    5: 'Παρασκευή',
+    6: 'Σάββατο',
+  },
+
+  months: {
+    0: 'Ιανουάριου',
+    1: 'Φεβρουάριου',
+    2: 'Μάρτιου',
+    3: 'Απρίλιου',
+    4: 'Μάιου',
+    5: 'Ιούνιου',
+    6: 'Ιούλιου',
+    7: 'Αύγουστου',
+    8: 'Σεπτέμβριου',
```

Comment:
[Wiktionary](https://en.wiktionary.org/wiki/%CE%A3%CE%B5%CF%80%CF%84%CE%B5%CE%BC%CE%B2%CF%81%CE%AF%CE%BF%CF%85#Greek) suggests to use `Σεπτεμβρίου` as Genitive.

```suggestion
    8: 'Σεπτεμβρίου',
```